### ISSUE TITLE
Bilingual data from Jun Ho

### ISSUE THREAD

**Issue by alvinwmtan** [langcog/wordbank#309]
Ready for import

**Comment by alvinwmtan** [comment]
Citation: 
Chai, J. H., Low, H. M., Wong, T. P., Onnis, L., & Mayor, J. (2021). Extra-linguistic modulation of the English noun-bias: Evidence from Malaysian bilingual infants and toddlers. Journal of Cultural Cognitive Science, 5(1), 49-64.

Contributor:
Jun Ho Chai, Chosun University

**Comment by HenryMehta** [comment]
@alvinwmtan Where is the data?

**Comment by alvinwmtan** [comment]
@HenryMehta data on the way, will update when it's ready

**Comment by alvinwmtan** [comment]
Processed data here (note three new languages: English (Malaysian), Malay (Malaysian), Mandarin (Malaysian)): 
[[English_Malaysian_WS].csv](https://github.com/langcog/wordbank/files/14751887/English_Malaysian_WS.csv)
[[Malay_Malaysian_WS].csv](https://github.com/langcog/wordbank/files/14751888/Malay_Malaysian_WS.csv)
[[Mandarin_Malaysian_WS].csv](https://github.com/langcog/wordbank/files/14751889/Mandarin_Malaysian_WS.csv)
[EnglishMalaysianWS_Chai_data.csv](https://github.com/langcog/wordbank/files/14751890/EnglishMalaysianWS_Chai_data.csv)
[EnglishMalaysianWS_Chai_fields.csv](https://github.com/langcog/wordbank/files/14751891/EnglishMalaysianWS_Chai_fields.csv)
[EnglishMalaysianWS_Chai_values.csv](https://github.com/langcog/wordbank/files/14751892/EnglishMalaysianWS_Chai_values.csv)
[MalayMalaysianWS_Chai_data.csv](https://github.com/langcog/wordbank/files/14751893/MalayMalaysianWS_Chai_data.csv)
[MalayMalaysianWS_Chai_fields.csv](https://github.com/langcog/wordbank/files/14751895/MalayMalaysianWS_Chai_fields.csv)
[MalayMalaysianWS_Chai_values.csv](https://github.com/langcog/wordbank/files/14751896/MalayMalaysianWS_Chai_values.csv)
[MandarinMalaysianWS_Chai_data.csv](https://github.com/langcog/wordbank/files/14751897/MandarinMalaysianWS_Chai_data.csv)
[MandarinMalaysianWS_Chai_fields.csv](https://github.com/langcog/wordbank/files/14751899/MandarinMalaysianWS_Chai_fields.csv)
[MandarinMalaysianWS_Chai_values.csv](https://github.com/langcog/wordbank/files/14751900/MandarinMalaysianWS_Chai_values.csv)
[MalaysianCDI_notes.md](https://github.com/langcog/wordbank/files/14751901/MalaysianCDI_notes.md)

**Comment by HenryMehta** [comment]
@alvinwmtan Do you want me to start on this?

**Comment by alvinwmtan** [comment]
@HenryMehta nope—I'm just working on them incrementally, but we expect that there's gonna be a bigger influx of datasets sometime around summer, so I would say hold off until then and we can push them through all at once. (Feel free to ignore the other files that I put up in other issues as well)

**Comment by alvinwmtan** [comment]
Imported in v3.0
